hypervisor/include/common/ptdev.h

```diff
@@ -19,6 +19,11 @@ enum intx_ctlr {
 #define PTDEV_INTR_MSI		(1U << 0U)
 #define PTDEV_INTR_INTX		(1U << 1U)
 
+#define GPU_OPREGION_SIZE	0x5000U
+#define GPU_OPREGION_GPA	0x40880000U
+#define PCIR_ASLS_CTL		0xfcU /* Opregion start addr register */
+#define PCIM_ASLS_OPREGION_MASK	0xfffff000U /* opregion need 4KB aligned */
```

Comment:
Why this is related to PCIM_ ?

Reply:
Hi, Eddie. I think it's memory-related, so I use `PCIM_`, Maybe I use the wrong prefix, any suggestion for the name? It stands for the page offset for Opregion base address, I will refine the comment too.

Reply:
By checking all the existing PCIM_XXX macros, It seems to be confusing... But I am fine to take for now.